roles/blackbox_exporter/defaults/main.yml

```diff
@@ -71,7 +71,10 @@ blackbox_exporter_configuration_modules:
 #      validate_answer_rrs:
 #        fail_if_matches_regexp: [test]
 
-# Where to put the blackbox_exporter.yml main configuration file
-blackbox_exporter_config_dir: /etc
+# Where to put the blackbox.yml main configuration file
+blackbox_exporter_config_dir: /etc/prometheus
```

Comment:
This should be `/etc/blackbox_exporter` to match the other roles.

Reply:
okay, done.